enhancements/update/change-management-and-maintenance-schedules.md

```diff
@@ -0,0 +1,1158 @@
+controlling exactly when changes are applied to critical systems is universally appreciated in enterprise 
+software. 
+
+Since these are such well established principles, I will summarize the motivation as helping
+OpenShift meet industry standard expectations with respect to limiting potentially disruptive change 
+outside well planned time windows. 
+
+It could be argued that rigorous and time sensitive management of OpenShift cluster API resources could prevent
+unplanned material changes, but Change Management / Maintenance Schedules introduce higher level, platform native, and more 
+intuitive guard rails. For example, consider the common pattern of a gitops configured OpenShift cluster.
+If a user wants to introduce a change to a MachineConfig, it is simple to merge a change to the
+resource without appreciating the fact that it will trigger a rolling reboot of nodes in the cluster.
+
+Trying to merge this change at a particular time of day and/or trying to pause and unpause a 
+MachineConfigPool to limit the impact of that merge to a particular time window requires
+significant forethought by the user. Even with that forethought, if an enterprise wants 
+changes to only be applied during weekends, additional custom mechanics would need
+to be employed to ensure the change merged during the weekend without needing someone present.
+
+Contrast this complexity with the user setting a Change Management / Maintenance Schedule on the cluster. The user
+is then free to merge configuration changes and gitops can apply those changes to OpenShift
+resources, but material change to the cluster will not be initiated until a time permitted
+by the Maintenance Schedule. Users do not require special insight into the implications of
+configuring platform resources as the top-level Maintenance Schedule control will help ensure
+that potentially disruptive changes are limited to well known time windows.
+
+#### Reducing Service Delivery Operational Tooling
+Service Delivery, as part of our OpenShift Dedicated, ROSA and other offerings is keenly aware of
+the issues motivating the Change Management / Maintenance Schedule concept. This is evidenced by their design
+and implementation of tooling to fill the gaps in the platform the preceding sections
+suggest exist.
+
+Specifically, Service Delivery has developed UXs outside the platform which allow customers 
+to define a preferred maintenance window. For example, when requesting an update, the user 
+can specify the desired start time. This is honored by Service Delivery tooling (unless
+there are reasons to supersede the customer's preference).
+
+By acknowledging the need for scheduled maintenance in the platform, we reduce the need for Service
+Delivery to develop and maintain custom tooling to manage the platform while 
+simultaneously reducing simplifying management for all customer facing similar challenges.
```

Comment:
I'm not an authority on its implementation, but I believe it has aspects spread across OCM, Hive, and MUO. I think the implementation would also differ for HCP.
Assuming we could take it over, it would not facilitate the separation of control-plane and worker-node updates since it only creates a window for an entire cluster update.